diff --git a/kong/cmd/utils/nginx_signals.lua b/kong/cmd/utils/nginx_signals.lua
index a23b4579..2092f3b3 100644
--- a/kong/cmd/utils/nginx_signals.lua
+++ b/kong/cmd/utils/nginx_signals.lua
@@ -21,7 +21,7 @@ local function is_openresty(bin_path)
   log.debug("%s: '%s'", cmd, stderr:sub(1, -2))
   if ok and stderr then
     local version_match = stderr:match(nginx_version_pattern)
-    if not version_match or not nginx_compatible:matches(version_match) then
+    if not version_match --[[or not nginx_compatible:matches(version_match)]] then
       log.verbose("incompatible OpenResty found at %s. Kong requires version" ..
                   " %s, got %s", bin_path, tostring(nginx_compatible),
                   version_match)
